apps/web/src/components/layout/components/v2/BridgeUpdateModal.tsx

```diff
@@ -33,14 +34,38 @@ export const BridgeUpdateModal: FC<BridgeUpdateModalProps> = ({ isOpen, toggleOp
     setUrl(event.target.value);
   };
 
+  const validateFromLocal = async (bridgeUrl: string): Promise<{ isValid: boolean }> => {
+    try {
+      const client = buildBridgeHTTPClient(bridgeUrl);
+      const response = await client.healthCheck();
+
+      const result = { isValid: response.status === 'ok' };
+
+      return result;
+    } catch {}
+
+    return { isValid: false };
+  };
+  const localDomains = ['localhost', '127.0.0.1'];
+  const isLocalAddress = () => {
+    return localDomains.includes(location.hostname);
+  };
+
   const onUpdateClick = async () => {
     setUrlError('');
     setIsUpdating(true);
     try {
       if (url) {
         validateURL(url);
 
-        const result = await validateBridgeUrl({ bridgeUrl: url });
+        let result =
+          isLocalStudio && isLocalAddress()
```

Comment:
we also need to support supplying the local tunnel address here as well as the local address